### ISSUE TITLE
Authentication broken for ancient Zabbix since support added for Zabbix 6.4

### ISSUE THREAD

**Issue by colinleroy** [grafana/grafana-zabbix#1616]
Zabbix changed their user.login API endpoint to use "username" instead of "user" in 6.4rc, and you added support for it.
You added a isDeprecatedUserParamError check in order to be retro-compatible, and you look for 'unexpected parameter "user' in the error string. So far so good.

But ancient Zabbix installs are now broken, because when getting a user.login request with "username" instead of "user", they throw a generic error: 

```
$ cat login.json; curl  --header 'Content-Type: application/json-rpc' -k https://zabbix-3.0-instance/zabbix/api_jsonrpc.php -d @login.json 
{
  "jsonrpc": "2.0",
  "method": "user.login",
  "params": {
    "username": "api_grafana",
    "password": "xxxxxxxxxxxxxxxxxxxxxxxxxxxxxx"
  },
  "id": 1
}

{"jsonrpc":"2.0","error":{"code":-32602,"message":"Invalid params.","data":"Login name or password is incorrect."},"id":1}
```

Sadly, I have no idea at what point Zabbix changed from that generic error to the one where they check the parameter names.

Possibly it would be cleaner to check the apiinfo.version endpoint before trying to user.login, and set the parameter name accordingly. Unfortunately, I suck at go and don't think I can provide a good pull request.

Hope this helps!

**Comment by theguimaraes** [comment]
I have the same issue here.

Grafana 9.4.3 and i can't connect to one of my old Zabbix 3.0

Plugin version 4.3.0 and 4.3.1, only connect to my new Zabbix 6.0, but can't connect to the 3.0

Plugin version 4.2.10 and lower, i can connect to my old Zabbix 3.0, but can't connect to the new 6.0. I receive a Error: Invalid params. Invalid parameter "/": unexpected parameter "user".

**Comment by renatotoniolo** [comment]
I'm having the same problem, my Zabbix 3.0 in which I'm migrating to a new Zabbix 6.4 in parallel, I'm using two datasources to keep the alerts in a single dashboard and I'm having the same problem.

**Comment by alexanderzobnin** [comment]
Fixed in 4.3 #1544

**Comment by colinleroy** [comment]
> Fixed in 4.3 #1544

No, this is a different issue. The 4.3 makes the plugin work on Zabbix 6.4, but breaks the plugin with Zabbix 3.0
